src/openads/infrastructure/connector/appnexus/AppNexusConnectorImpl.js

```diff
@@ -80,11 +82,21 @@ export default class AppNexusConnectorImpl extends AppNexusConnector {
   }
 
   refresh (target) {
-    this._logger.debug('Refresh AppNexus Tag', '| target:', target)
-    this._appNexusClient.anq.push(() => this._appNexusClient.refresh(target))
+    this._logger.debug('Refresh has been requested')
+    if (this._buffer !== undefined) clearTimeout(this._buffer)
+    this._bufferAccumulator.push(target)
+    this._refreshBufferOperator()
     return this
   }
 
+  _refreshBufferOperator () {
+    this._buffer = setTimeout(() => {
+      this._logger.debug('Refresh is called')
+      this._appNexusClient.anq.push(() => this._appNexusClient.refresh(this._bufferAccumulator))
+      this._buffer = undefined
```

Comment:
When is _bufferAccumulator cleaned after _refreshBufferOperator method is called?